Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -2394,3 +2401,29 @@ isGoodRelativeDirectoryPath = state0
 --     | otherwise           -> 4
 -- 6 -> \_ -> 6 -- black hole
 -- @
+
+-- TODO: move to Distribution.Version
```

Comment:
Why the TODO?

Reply:
Oh, great point! In fact there is already such a function there. Fixed.

Reply:
@emilypi maybe you could take a look and say if it's good enough now. 🙏